install.sh

```diff
@@ -69,7 +69,7 @@ install_nvm_from_git() {
   if [ -d "$INSTALL_DIR/.git" ]; then
     echo "=> nvm is already installed in $INSTALL_DIR, trying to update using git"
     printf "\r=> "
-    cd "$INSTALL_DIR" && (command git fetch 2> /dev/null || {
+    pushd "$INSTALL_DIR" && (command git fetch 2> /dev/null || {
       echo >&2 "Failed to update nvm, run 'git fetch' in $INSTALL_DIR yourself." && exit 1
```

Comment:
I'm beginning to think that we could get rid of `cd` and friends altogether.  Instead use the `--git-dir` and `--work-tree` options for the `git` commands.

What do you think?